apps/frontend-nuxt/pages/blog/index.vue

```diff
@@ -0,0 +1,36 @@
+<script setup lang="ts">
+import { BlogRsc } from "@effect-ts-app/boilerplate-client"
+
+const blogClient = clientFor(BlogRsc)
+
+const [, createPost_] = useMutation(blogClient.createPost)
```

Comment:
Where do `useMutation` and `useSafeQuery` come from?

Reply:
Click them and find out.